components/messages.json

```diff
@@ -10,6 +10,11 @@
       "#normal": {
         "text": "@theme-text-secondary-normal"
       }
+    },
+    "extParticipants": {
+      "#normal": {
+        "text": "@theme-text-warning-normal"
+      }
```

Comment:
why is there no hver or pressed state in normal but there is in high contrast?

Reply:
![image](https://user-images.githubusercontent.com/27616697/164681198-9f416a24-54ab-4893-ab4c-8aa89d9b2d9f.png)

This is the color token for this domain name which remains the same color on hover/pressed in non-HC themes. However in High Contrast, when hovered over a message, the background changes to sky blue, so we need the domain name to turn black for visibility.

Similar is the case for the `globalHeader-buttonDevice-text-active`.